aws-hybrid-vpn-gcp/inputs.tf

```diff
@@ -0,0 +1,30 @@
+variable "name" {
+  description = "Name of the Hybrid VPN deployment, used in various resources to uniquely identify them"
+  type        = string
+  default     = "test"
+}
+
+variable "aws_vpc_id" {
+  description = "AWS VPC to connect the VPN to"
+  type        = string
+}
+
+variable "aws_subnet_ids" {
+  description = "AWS subnets to connect the VPN to"
+  type        = list(string)
+}
+
+variable "gcp_network_name" {
+  description = "Name of the GCP VPC network to connect the VPN to"
+  type        = string
+}
+
+variable "gcp_network_id" {
```

Comment:
How come we use `network` for GCP but `vpc` for AWS? I would lean towards consistency unless there's a strong reason to do otherwise.